geth/jail/cell_test.go

```diff
@@ -75,14 +77,19 @@ func (s *CellTestSuite) TestCellLoopRace() {
 	}
 }
 
-// TestJailFetchRace tests multiple fetch callbacks,
-// supposed to be run with '-race' flag.
+// TestJailFetchRace tests multiple sending multiple HTTP requests simultaneously in one cell, using `fetch`.
+// Supposed to be run with '-race' flag.
 func (s *CellTestSuite) TestCellFetchRace() {
-	body := `{"key": "value"}`
+	// How many request should the test perform ?
+	const requestCount = 5
```

Comment:
does it make sense to move these variables lower (right above where they are used)?

Reply:
Moved the `expected` down above the loop. Do you prefer the constants to be defined more below, too ?

Reply:
would be easier to understand, but i don't have strong preference here